pythonforandroid/bootstraps/common/build/build.py

```diff
@@ -326,11 +326,28 @@ def make_package(args):
         args.icon or default_icon,
         join(res_dir, 'drawable/icon.png')
     )
+
     if get_bootstrap_name() != "service_only":
-        shutil.copy(
-            args.presplash or default_presplash,
-            join(res_dir, 'drawable/presplash.jpg')
-        )
+        lottie_splashscreen = join(res_dir, 'raw/splashscreen.json')
+        if args.presplash_lottie:
+            shutil.copy(
+                'templates/lottie.xml',
+                join(res_dir, 'layout/lottie.xml')
+            )
+            ensure_dir(join(res_dir, 'raw'))
+            shutil.copy(
+                args.presplash_lottie,
+                join(res_dir, 'raw/splashscreen.json')
+            )
+        else:
+            if exists(lottie_splashscreen):
+                remove(lottie_splashscreen)
+                remove(join(res_dir, 'layout/lottie.xml'))
```

Comment:
We could start using pathlib. Or maybe next PR :sweat_smile:

Reply:
yep, though about it as well, but decided to leave it for later.